.github/actions/publish-nuget/New-NuGetPackage.ps1

```diff
@@ -0,0 +1,32 @@
+<#
+.SYNOPSIS
+    Creates a NuGet package from each project in the sln file in the current directory.
+.DESCRIPTION
+    Uses "dotnet sln list" to get all projects in the current directory. This means the current directory must have 
+    exactly one sln file in it. Then calls "dotnet pack" for each csproj file with the provided arguments. If there is
+    a nuspec file in the project's directory too, then it is used to generate the package description instead of the 
+    regular auto-generation.
+.EXAMPLE
+    New-NugetPackage @("--configuration:Release", "--warnaserror")
+    Calls "dotnet pack project.csproj --configuration:Release --warnaserror" on each project.
+#>
+
+param([array] $Arguments)
+
+# We go through the projects individually, because the "-p:NuspecFile=" parameter can't be passed to a solution.
```

Comment:
Since this is the main reason for having this script in the first place, please add this information to the DESCRIPTION.

Reply:
I will do it because I'm sick of this issue and want to get it over with already. But I also disagree. While this is the technical reason to justify a script, from an invoker perspective it's only an implementation detail.